pkg/ocp/nodesdebug/nodes_debug.go

```
@@ -0,0 +1,344 @@
+	}
+	sanitized := strings.Trim(b.String(), "-")
+	if sanitized == "" {
+		sanitized = "node"
+	}
+	if len(sanitized) > 40 {
+		sanitized = sanitized[:40]
+	}
+	return sanitized
+}
+
+func containerStatusByName(statuses []corev1.ContainerStatus, name string) *corev1.ContainerStatus {
```

golangci-lint (2.12.2)

[error] func containerStatusByName is unused

(unused)